src/trials.jl

```diff
@@ -8,9 +8,12 @@ mutable struct Trial
     gctimes::Vector{Float64}
     memory::Int
     allocs::Int
```

Comment:
To limit how breaking this is, e.g. https://github.com/PTsolvers/HPCBenchmarks.jl/blob/a5985aaaf931efb0caf194d669e3bfcb90c5c08e/CUDA/common.jl#L24 would break, a constructor could be added which defaults `linux_perf_stats` to be nothing.